challenges/classes.js

```diff
@@ -1,7 +1,28 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
-
+class cuboidMaker {
```

Comment:
Its not necessary but it is an understood standard to name classes with uppercases, in this case: CuboidMaker